Add Data Analytics and Backend sections (#5877)
DevExpress/devextreme-documentation: concepts/Common/Data Analytics and BI/Embedded Reports/Embedded Reports.md

> Explore [DevExpress Reporting](https://www.devexpress.com/subscriptions/reporting/) - a reporting solution designed to meet your needs for advanced reporting functionality, rich design-time experience, and extensive customization options.

Reviewer: ```suggestion
[DevExpress Reporting](https://www.devexpress.com/subscriptions/reporting/) is a reporting solution designed for advanced reporting, and includes a rich design-time experience, and extensive customization options.
```

Not sure what `a rich design-time experience` and `customization options` imply

> Explore [DevExpress Reporting](https://www.devexpress.com/subscriptions/reporting/) - a reporting solution designed to meet your needs for advanced reporting functionality, rich design-time experience, and extensive customization options.

Author: ```suggestion
[DevExpress Reporting](https://www.devexpress.com/subscriptions/reporting/) is a reporting solution designed for advanced reporting, and includes a rich design-time experience and extensive customization options.
```

![DevExpress Reporting](/images/Common/Embedded_Reports.png)

#### Key Features

- Subreports, master-detail reports, report parameters, expressions, data bindings, and conditional formatting.

> - An intuitive drag-and-drop interface and [integration with Visual Studio](https://docs.devexpress.com/XtraReports/404722/vs-code-report-designer).

Reviewer 2: Let's add VS code also:
An intuitive drag-and-drop interface and integration with [Visual Studio](https://docs.devexpress.com/XtraReports/404722/vs-code-report-designer) and VS Code (link).

- [End-User Report Designer](https://docs.devexpress.com/XtraReports/404503/get-started-with-devexpress-reporting/create-an-end-user-designer) to create and edit reports at runtime.

> - [Web Document Viewer](https://docs.devexpress.com/XtraReports/403569/web-reporting/common-features/devexpress-document-viewers) includes an interactive preview of generated documents, allowing users to view, print, and export reports.

Reviewer 2: You wrote every list item without a verb, let's do the same here.
```suggestion
- [Web Document Viewer](https://docs.devexpress.com/XtraReports/403569/web-reporting/common-features/devexpress-document-viewers) with an interactive preview of generated documents, allowing users to view, print, and export reports.
```

> DevExpress Reporting is compatible across various platforms including ASP.NET WebForms, ASP.NET MVC, ASP.NET Core, Blazor, and JavaScript applications based on Angular, React, and Vue frameworks.

Reviewer 3: I would reorder the platforms to reflect the context. First mention JS frameworks, then ASP.NET Core. The legacy stuff I would mention separately (just in case).

> [note] You can select from a range of pricing options, including standalone Reporting subscriptions or the Universal package, which includes both DevExpress Reporting and DevExtreme.

Reviewer 3: Let's make subscription mentions to be links to the corresponding pages (as in other sections)

> [note] You can select from a range of pricing options, including standalone [Reporting subscriptions](https://www.devexpress.com/subscriptions/reporting/) or the [Universal package](https://www.devexpress.com/subscriptions/universal.xml), which includes both DevExpress Reporting and DevExtreme.

Reviewer: ```suggestion
[note] You can select from a range of pricing options, including standalone [Reporting subscriptions](https://www.devexpress.com/subscriptions/reporting/) or the [Universal package](https://www.devexpress.com/subscriptions/universal.xml) that includes DevExpress Reporting and DevExtreme.
```
